# dist_mat callback

---
custom_msgs/matrix[] dist_mat
